DS13/code/modules/overmap/escape_pod.dm

```diff
@@ -0,0 +1,224 @@
+	playsound(src, 'sound/effects/footstep/catwalk5.ogg', 100, 1)
+
+/obj/structure/escape_pod_ladder/attack_ai(mob/user)
+	if(!linked)
+		link_up()
+		return
+	var/mob/living/silicon/ai/A = user
+	if(A.eyeobj)
+		to_chat(user, "<span class='notice'>You move your camera down [src].</span>")
+		A.eyeobj.forceMove(get_turf(linked))
+
+/obj/structure/escape_pod_ladder/exit
+	name = "Escape pod exit hatch"
+	desc = "A ladder which allows you to exit an escape pod. It will automatically retract once the escape pod is launched."
+	icon_state = "ladder10"
+
+/obj/structure/escape_pod_ladder/exit/two
+	name = "Escape pod exit hatch"
+	desc = "A ladder which allows you to exit an escape pod. It will automatically retract once the escape pod is launched."
+	icon_state = "ladder10"
+	class = "escapepod2"
+
+/obj/structure/escape_pod_ladder/Initialize(mapload)
+	. = ..()
+	GLOB.escape_pod_objects += src
+	addtimer(CALLBACK(src, .proc/link_up), 60)
+
+/obj/structure/escape_pod_ladder/proc/link_up()
+	for(var/X in GLOB.escape_pod_objects)
+		if(istype(X, /obj/structure/escape_pod_ladder/exit))
+			var/obj/structure/escape_pod_ladder/exit/partner = X
+			if(!linked && !partner.linked && partner.class == class)
+				linked = partner
+				partner.linked = src
+
+/obj/structure/escape_pod_ladder/exit/link_up()
+	return
+
+/obj/structure/overmap_component/helm/escape_pod/examine(mob/user)
+	. = ..()
+	if(!GLOB.pods_armed)
+		to_chat(user, "<span class='warning'>It is currently locked, as the escape pods are not armed.</span>")
+
+/area/ship/escape_pod
+	name = "Escape pod 1"
+	class = "escapepod1"
+	has_gravity = STANDARD_GRAVITY
+	explosion_exempt = TRUE
+	looping_ambience = 'DS13/sound/ambience/escape_pod.ogg'
+
+/area/ship/escape_pod/two
+	name = "Escape pod 2"
+	class = "escapepod2"
+	has_gravity = STANDARD_GRAVITY
+	explosion_exempt = TRUE
+
+/area/ship/escape_pod/three
+	name = "Escape pod 3"
+	class = "escapepod3"
+	has_gravity = STANDARD_GRAVITY
+	explosion_exempt = TRUE
+
+/obj/structure/overmap_component/helm
+	name = "Piloting station"
+	desc = "This console gives you the power to control a starship. Screwdriver it to change its command codes."
+	icon_state = "pilot"
+	position = "pilot"
+	req_access = list(ACCESS_HEADS)
+	flags_1 = HEAR_1
+
+/obj/machinery/computer/communications/proc/arm_pods(var/mob/living/carbon/user)
+	if(GLOB.pods_armed)
+		var/question = alert("The escape pods are already armed. Disarm them?",name,"Yes","No")
+		if(question == "No" || !question)
+			return
+		GLOB.pods_armed = FALSE
+		priority_announce("Escape pods have been disarmed. All hands should return to their stations.","Escape pods disarmed",'sound/ai/commandreport.ogg')
+		var/obj/structure/overmap_component/helm/HH = locate(/obj/structure/overmap_component/helm) in get_area(src)
+		if(HH)
+			HH.redalert_end()
+		return
+	var/question = alert("Arm the escape pods for emergency evacuation?.",name,"Yes","No")
+	if(question == "No" || !question)
```

Comment:
Technically we're doing a check on the answer, not the question. Perhaps the return value of alerts should be a `var/answer`?